packages/abap-deploy-config-sub-generator/src/app/types.ts

```diff
@@ -0,0 +1,64 @@
+import type { AbapDeployConfigAnswersInternal } from '@sap-ux/abap-deploy-config-inquirer';
+import type { AppWizard } from '@sap-devx/yeoman-ui-types';
+import type { ConnectedSystem } from '@sap-ux/deploy-config-generator-shared';
+import type { TelemetryData } from '@sap-ux/fiori-generator-shared';
+
+export interface AbapDeployConfigOptions extends AbapDeployConfigAnswersInternal {
+    /**
+     * VSCode instance
+     */
+    vscode?: unknown;
+    /**
+     * AppWizard instance
+     */
+    appWizard?: AppWizard;
+    /**
+     * Whether the generator is launched as a subgenerator
+     */
+    launchDeployConfigAsSubGenerator?: boolean;
+    /**
+     * Whether the generator is launched as a standalone generator
+     */
+    launchStandaloneFromYui?: boolean;
+    /**
+     * Path to the project
+     */
+    destinationRoot?: string;
+    /**
+     * Path to the project root
+     */
+    projectPath?: string;
```

Comment:
Whats the difference between `destinationRoot` //Path to the project and `projectPath` // Path to the project root

Reply:
I've consolidated these options into `projectPath`.